config/images/images.yaml

```diff
@@ -44,13 +44,18 @@ images:
   - v2.1.4
   - v2.2.0
   - v2.2.2
+- source: ghcr.io/fluent/fluent-operator/fluent-bit # A custom Fluent Bit image is required to work with FluentBit Operator for dynamic configuration reloading, ref: https://github.com/fluent/fluent-operator#fluent-bit
+  destination: europe-docker.pkg.dev/gardener-project/releases/3rd/fluent-operator/fluent-bit
+  tags:
+  - v3.0.7
```

Comment:
Before we where using the `kubesphere` image for this case ([ref](https://github.com/gardener/ci-infra/blob/403d6babe3798c7876e9ec3c9d102254630991be/config/images/images.yaml#L39-L46)) which has a similar comment.
Is there a special reason for the change? If so, please adapt the comment of this image to remind people not to use it anymore.

Reply:
Thanks, the feedback is applied.